.github/workflows/main.yml

```diff
@@ -114,6 +114,56 @@ jobs:
           # would be no failed golden test to upload.
           if-no-files-found: ignore   
   
+  ios-integration-test:
+    runs-on: macos-latest
+    timeout-minutes: 45
+    strategy:
+      matrix:
+        device:
+          - "iPhone 13"
```

Comment:
Should we add more than the iPhone 13? Our project can run 5 macOS build simultaneously.

Reply:
Hm, idk. What do you think?

Reply:
If you want you can add more. I guess it might help to see if were breaking old iOS devices?

Reply:
I think one iPhone is for the beginning enough. We need also for the Android integration tests macOS machines, so I think we should not unnecessarily builds. If there is a reason to test on more OS version, then let's do this.